.circleci/config.yml

```diff
@@ -35,39 +35,52 @@ jobs:
     steps:
       - checkout
       - run:
-          name: "Pull Submodules"
+          name: "Checkout Submodules"
           command: |
-            git submodule init
-            git submodule update --remote
+            git submodule update --init
+      - run:
+          name: Add Docker registry to /etc/hosts
+          command: echo "172.30.1.1 docker-registry.default.svc" | sudo tee -a /etc/hosts
       - redhat-openshift/create-local-cluster-with-oc:
           skip-registry-check: true
+      - run:
+          command: docker login -u "$RED_HAT_DOCKER_USERNAME" -p "$RED_HAT_DOCKER_PASSWORD" "$RED_HAT_DOCKER_SERVER"
+          name: Log in to Red Hat docker registry
+      - run:
+          name: Login as cluster admin
+          command: oc login -u system:admin
+      - run:
+          command: make mock_storageclass
+          name: Mock the storage classes
       - redhat-openshift/login-and-update-kubeconfig:
           insecure-skip-tls-verify: true
           openshift-platform-version: 3.x
           password: password
           server-address: 'https://127.0.0.1:8443'
           username: dev1
-      - run:
-          command: docker login -u "$RED_HAT_DOCKER_USERNAME" -p "$RED_HAT_DOCKER_PASSWORD" "$RED_HAT_DOCKER_SERVER"
-          name: Log in to Red Hat docker registry
       - run:
           command: make provision
           name: Provision the cluster project set
+      - run:
+          command: |
+            rm -rf cas-postgres
+            git clone --single-branch --branch shipit-deploy/wksv3k-dev https://github.com/bcgov/cas-postgres.git
+            pushd cas-postgres || exit 1
+            git submodule update --init
+            make configure GIT_SHA1=`git rev-parse HEAD`
+            make build GIT_SHA1=`git rev-parse HEAD`
```

Comment:
Why do you need to specify the GIT_SHA1 here?

Reply:
In `git.mk` we have `GIT_SHA1=$(shell echo "$${CIRCLE_SHA1:-`$(GIT) rev-parse HEAD`}")` which gives precedence to the circle env var over getting the hash from git. We could just change that...

Reply:
see bcgov/cas-pipeline#18

Reply:
That PR is now merged, you can remove the GIT_SHA1 from here, and update the submodule to https://github.com/bcgov/cas-pipeline/commit/fced66d2ff08fb7f4c327e0cecf29ae4ef150e89